plugins/faqs/enhance.js

```diff
@@ -0,0 +1,41 @@
+function slugify(input) {
+  return String(input)
+    .trim()
+    .toLowerCase()
+    .replace(/([^\w]|\s)+/g, '-')
+    .replace(/-$/g, '')
+}
+
+export default ({ Vue, siteData }) => {
+  Vue.mixin({ methods: { slugify } })
```

Comment:
Do you think we need it everywhere?

Reply:
No, definitley not.

I am not sure how to mix the method in to only `FAQItem.vue` - I will research it!

Reply:
Just import it and add as a method.

Reply:
Done.